falconapp/views.py

```diff
@@ -1,4 +1,5 @@
 from django.shortcuts import render
 
+
 def login(request):
-    return render(request, 'login.html')
\ No newline at end of file
+    return render(request, "falconapp/login.html")
```

Comment:
under falconapp there is no login.html. 
please fix it

Reply:
it's a little confusing why you need falconapp directory if you have already a falcon directory?

Reply:
The problem is that I name it 'falconapp' or the fact that there are both 'falcon' and 'falconapp' directories? we were asked from barak to change what we did before and create a django application that holds the login.
All of the changes were taken as asked in the first challenge (but with name changes), including the directories names and directories tree structure.

Reply:
so maybe change it to falcon_django_apps?

Reply:
please fix the flake8 issues, maybe put those rows with #

Reply:
./falconapp/models.py:1:1: F401 'django.db.models' imported but unused
./falconapp/tests.py:1:1: F401 'django.test.TestCase' imported but unused
./falconapp/admin.py:1:1: F401 'django.contrib.admin' imported but unused